xrpl-wasm-std/src/core/types/opaque_float.rs

```diff
@@ -25,7 +25,7 @@
 /// # Example
 ///
 /// ```no_run
-/// # use xrpl_wasm_std::core::types::amount::opaque_float::OpaqueFloat;
+/// # use xrpl_wasm_std::core::types::opaque_float::OpaqueFloat;
```

Comment:
Before we make this change, we should decided if the `types` module contains only rippled-types or not. Thoughts?